docs/ca/dev/preparation-for-going-live.md

```diff
@@ -58,76 +54,131 @@ Double check that you do not have any clear text passwords stored in config file
+
 ## Two weeks before go-live ##
 
-- *Code freeze*. We recomment to have a code freeze at least two weeks before going live.
+- *Code freeze*. We recommend to have a code freeze at least two weeks before going live.
 - *Double-check the go-live date*. If any of the preceding tasks are not complete, postpone your go-live or discuss with us how to complete them in time. DNS changes are especially sensitive to deadlines. Due to how the DNS system works, any DNS changes take time to take effect.
 - *Make sure that the rollback strategy is still valid*. Check that you have everything you need to recover from an unforeseen issue with the newest version of the project you are deploying.
-- *Validate testing strategy.*
-- *Organize a go-live support team*. Prepare a team that can monitor your go-live, react quickly to any issues, and work with the Spryker Support or Operations teams.
-- *Validate BI and analytics integrations*. They should not be connected to your production database, but rather to the provided read replica. Make sure no external system is interacting with your production database.
+- *Make sure that DNS is set*.
 - *Remove all the demo data from the environment*. The project should only use the real data that will be used after the go-live. Remove all the demo data that comes with the Spryker repository, which includes demo and admin users. Demo users in a live shop pose a significant security risk for your project.
+- *Make sure that real data is set on production*
+  - Categories
+  - Product and all related data (prices, offers, attributes, images, stock, labels, reviews and ratings, cross and up selling etc.)
+  - Customers, Companies, Users, Merchants
+  - Taxes
+  - Discounts
+  - CMS pages (including homepage, CSM blocks, final terms & conditions check)
+  - Glossary (including all the needed languages)
+  - Custom data
+  - Other data
+- *Make sure that 3rd party systems are switched to "Production mode"*
+  - *Set up production configuration for all the 3rd party systems* (in environment variables). Make sure to not expose secrets in the codebase.
+  - *Validate BI and analytics integrations*. They should not be connected to your production database, but rather to the provided read replica. Make sure no external system is interacting with your production database.
+- *Organize a go-live support team*. Prepare a team that can monitor your go-live, react quickly to any issues, and work with the Spryker Support or Operations teams.
 - *Define the exact plan for the go-live day.*:
   - Define the time of deployment.
   - Define the exact steps to be performed (including running Jenkins or other scripts if needed).
-- Make sure that DNS is set.
+- *Go live*: 
+  - *Prepare a go live communication plan*. Develop a communication plan to inform stakeholders, customers, and support teams about the launch date and any changes or updates.
+  - Remove basic auth from the frontend part and deploy the change.
```

Comment:
As their Go-Live date will be after 2 weeks, do you suggest this removal before 2 weeks ?

Reply:
Added a go-live section. Moved there